packages/@aws-cdk/aws-servicecatalog/README.md

```diff
@@ -35,6 +35,7 @@ enables organizations to create and manage catalogs of products for their end us
 - [Constraints](#constraints)
   - [Tag update constraint](#tag-update-constraint)
   - [Notify on stack events](#notify-on-stack-events)
+  - [Constrain Cloudformation parameters](#constrain-cloudformation-parameters)
```

Comment:
Let's call this "CloudFormation parameters constraint ", to be consistent with the other entries.